.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: Integration Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *"
+  push:
+    tags:
+      - "*"
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  ALLOW_PLOTTING: true
+  SHELLOPTS: 'errexit:pipefail'
+
+jobs:
+  # For now this is just MNE-Python, but others could be added
+  mne:
+    name: MNE-Python
+    runs-on: ubuntu-22.04
+    env:
+      DISPLAY: ':99.0'
+      MNE_LOGGING_LEVEL: 'info'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: git clone --depth=1 https://github.com/mne-tools/mne-python.git --branch main --single-branch
+      - run: ./mne-python/tools/setup_xvfb.sh
+      - name: Install MNE dependencies
+        run: ./tools/github_actions_dependencies.sh
```

Comment:
@larsoner, this scripts concerns me a bit. I'm noticing that it installs a lot of things at their latest development version. I imagine that this might lead to a lot of false flags here if something fails due to another dependency and not PyVista.

Reply:
It does install a bunch of other stuff. But:

1. It actually has been very stable for the last couple of years we've done it this way -- other than PyVista issues, most failures are with mpl deprecations, which we won't hit with the subset of tests in the `pytest` line here
2. There are some dependencies we need to get all visualizations, such as `nibabel` for MRI support, etc.

I'll trim it down to the minimal set of things we need instead of installing everything.

I'll also remove the failing `pyvista.Report` line for now. We can add it back later in a separate PR that fixes scooby or whatever needs to be done there...

Reply:
I think this just showed up here: https://github.com/pyvista/pyvista/runs/8142770010?check_suite_focus=true#step:10:1

Reply:
Okay, thanks for clarifying. I think this is close to done now